src/data/arbeidsuforhet/useSendForhandsvarsel.ts

```diff
@@ -14,11 +14,10 @@ export const useSendForhandsvarsel = () => {
 
   return useMutation({
     mutationFn: postForhandsvarsel,
-    onSuccess: (data) => {
-      queryClient.setQueryData(
-        arbeidsuforhetQueryKeys.arbeidsuforhet(personident),
-        data
-      );
+    onSuccess: () => {
+      return queryClient.invalidateQueries({
```

Comment:
Hadde det fungert å gjøre noe sånt for oppdatere querydata fra responsen:
``
    onSuccess: (data) => {
      queryClient.setQueryData(
        arbeidsuforhetQueryKeys.arbeidsuforhet(personident),
        (oldData: VurderingResponseDTO[]) => [data, ...oldData]
      );
    }
``

Reply:
Det kan sikkert funke! Men jeg er usikker på verdien av å innføre mer logikk her bare for å slippe å gjøre et enkelt kall til backend 🤔

Reply:
Neida, er ikke så viktig her så vi kan godt gå for invalidate :slightly_smiling_face: